docs/dev/tutorials/cross-chain-tutorial.md

```diff
@@ -575,6 +575,6 @@ The counter value is 1
 
 ## Learn more
```

Comment:
Not sure what is wrong with this text or github displaying it, but I see the following diff
<img width="1220" alt="image" src="https://github.com/matter-labs/zksync-web-era-docs/assets/52464764/1bc2a1e8-b503-4f64-b469-17d04f90eaed">

Reply:
yeah, weird glitch, it's fine